flink-filesystems/flink-s3-fs-hadoop/src/main/java/org/apache/flink/fs/s3hadoop/S3FileSystemFactory.java

```diff
@@ -51,7 +51,8 @@
 
 	private static final String[][] MIRRORED_CONFIG_KEYS = {
 			{ "fs.s3a.access-key", "fs.s3a.access.key" },
-			{ "fs.s3a.secret-key", "fs.s3a.secret.key" }
+			{ "fs.s3a.secret-key", "fs.s3a.secret.key" },
+			{ "fs.s3a.endpoint", "fs.s3a.endpoint" }
```

Comment:
I don't understand why we need to add this config key to the list of mirrored keys.
Isn't the purpose of the mirroring to map between Hadoop's keys and internal keys (of say Presto)?

If I'm commenting out this change, the new tests are still passing (which makes sense, because the config keys are just forwarded)

Reply:
You're right, I think I made a mistake thought that only keys listed in `MIRRORED_CONFIG_KEYS` would be added to Hadoop's configuration. This modification may not be necessary, sorry to bother.